kitchen.yml

```diff
@@ -4,31 +4,44 @@
 ---
 driver:
   name: docker
-  provision_command: curl -L https://www.chef.io/chef/install.sh | bash
+  # provision_command: curl -L https://www.chef.io/chef/install.sh | bash
 
 transport:
   name: docker
 
 provisioner:
   name: dummy
 
+verifier:
+  name: inspec
+  inspec_tests:
+    - test/integration/inspec
+
 platforms:
   - name: amazonlinux-2
-  - name: ubuntu-18.04
+  # - name: amazonlinux-2023
   - name: ubuntu-20.04
+  - name: ubuntu-22.04
+  - name: ubuntu-24.04
   - name: fedora-latest
     driver:
       provision_command:
         - yum install libxcrypt-compat -y
         - curl -L https://www.chef.io/chef/install.sh | bash
-  - name: centos-7
+  - name: centos-stream-9
+    driver:
+      image: dokken/centos-stream-9
   - name: oraclelinux-7
-  - name: rockylinux-8
-  - name: debian-11
+  - name: oraclelinux-8
+  - name: oraclelinux-9
+  - name: rockylinux-9
   - name: debian-12
-  - name: opensuse-15
+  - name: opensuse-15.5
+    driver:
+      image: opensuse/leap:15.5
+  - name: opensuse-15.6
     driver:
-      image: opensuse/leap:15
```

Comment:
This slug should get you the latest without having to maintain the version here

Reply:
I ended up putting in the two versions separately because both of them showed to still be under active support on the website.

Reply:
Oh wait I see what you mean. Specify a "latest v15" in addition to a "specific v15."

Reply:
Leap:15 should always be the latest